epub33/errata.html

```diff
@@ -0,0 +1,259 @@
+        <dt>Link to all errata:</dt>
+        <dd><span id="errata_link"></span></dd>
+      </dl>
+
+      <section data-notoc>
+        <h1>How to Submit an Erratum?</h1>
+        <p>Errata are introduced and stored in the <a href="https://github.com/w3c/epub-specs/issues/">issue list of the group‘s GitHub repository</a>. The workflow to add a new erratum is as follows:</p>
+        <ul>
+           <li>An issue is raised for a possible erratum. The label of the issue SHOULD be set to “<code>ErratumRaised</code>”. It SHOULD also include the label corresponding to the document on which the erratum is raised, e.g., “<code>Spec-EPUB3</code>” or “<code>Spec-ReadingSystems</code>” (please, consult the <a href="https://github.com/w3c/epub-specs/labels?q=spec">list of available labels</a>; look for labels of the form “Spec-”). It is o.k. for an erratum to have several labels. In some, exceptional, cases, i.e., when the erratum is very general, it is also acceptable not to have a reference to a document.</li>
+           <li>Issues labeled as “<code>Editorial</code>” are displayed separately, to make it easier to differentiate editorial errata from substantial ones.</li>
+          <li>The community discusses the issue. If it is accepted as a genuine erratum, the label “<code>Errata</code>” is added to the entry and the “<code>ErratumRaised</code>” label should be removed. Additionally, a new comment on the issue MAY be added, beginning with the word "Summary:" (if such a summary is useful based on the discussion).</li>
+          <li>All issues, labeled by “<code>Errata</code>”, are displayed in this report, whether they are opened or closed. Their status is added to the report on the individual errata.</li>
+          <li>If the community rejects the issue as an erratum, the issue should be closed.</li>
+          <li>ALL substantive errata are generally expected to have corresponding tests (such as a pull request in the <a href='https://github.com/w3c/epub-tests/'>EPUB 3 Test Repository</a>) either in the form of new tests or modifications to existing tests, or must include the rationale for why test updates are not required for the erratum.</li>
+      </ul>
+
+        <p>This report contains a reference to all open issues with the label <code>Errata</code>, displayed in the sections below. Each section collects the issues for a specific document, with a separate section for the issues not assigned to any.</p>
+
+        <p>If you have problems following this process, but you want nevertheless to report an error, you can also contact the staff contact of the Working Group, <a href="mailto:ivan@w3.org">Ivan Herman</a>.</p>
+      </section>
+    </header>
+
+    <div class="toc" id="toc"></div>
+
+    <main>
+      <!-- The data-erratalabel should include one label that filters the errata -->
+      <section data-erratalabel="Spec-Overview">
+        <h1>Open Errata on the “EPUB 3 Overview” Working Group Note</h1>
```

Comment:
Do we need errata for notes? Isn't it faster to just fix and republish them since they don't have the same involved update process as REC-track documents?

(Just thinking it might make for a less busy page.)

Reply:
I am a bit neutral about it... and would like to hear the reaction of the chairs.

My only reason to include it (apart from the fact that it does not cost anything to do it:-) is that, I presume, the new WG will not meet that frequently, so a number of errata may pile up between two calls, and this mechanism may help us keeping track of those.

Reply:
> the new WG will not meet that frequently, so a number of errata may pile up between two calls

Doesn't the WG have to agree to publish errata, though? I thought the sticking point for REC-track documents was getting them republished, so publishing errata is quicker, whereas notes we can freely revise and republish once we know what needs changing?

Reply:
It is all true. But will we look at the possible errata frequently? That is really the issue.

Again, I am not against removing the note references, but I would like to see how Wendy & Co imagine running the show...

Reply:
The only notes I see potentially receiving attention are the accessibility related ones, particularly as we approach EAA. I'm ok with errata and keeping track, but we can always revise the process if it becomes arduous?

Reply:
Yes, we can touch that whenever we want.

So let us leave it as is now.